<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>616</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="lblSalida">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>631</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>8514oem</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(185, 161, 255, 255), stop:1 rgba(255, 255, 255, 255));
font: 20pt &quot;8514oem&quot;;
padding-left: 3px;
padding-bottom: 2px;
border:2px solid;
border-radius: 8px</string>
   </property>
   <property name="text">
    <string>TextoPrueba</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TXTpWD">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>721</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(185, 161, 255, 255), stop:1 rgba(255, 255, 255, 255));
padding-left: 5px;
border: 2px solid;
border-radius: 20px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPwd">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(185, 161, 255, 255), stop:1 rgba(255, 255, 255, 255));
border: 2px solid;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>CONVERTIR A EMOJI</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSalida_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>470</y>
     <width>391</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>8514oem</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(24,218, 33);
font: 18pt &quot;8514oem&quot;;</string>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
